kpi/models/import_export_task.py

```diff
@@ -82,8 +82,9 @@ class Meta:
     )
 
     user = models.ForeignKey('auth.User', on_delete=models.CASCADE)
-    data = JSONField()
-    messages = JSONField(default={})
+    # ToDo validate whether both fields below need to be changed to jsonbfield
```

Comment:
`data` is definitely an improvement as shown in your `filter()` change below. as for `messages`, why not?